src/js/multiple-choice.js

```diff
@@ -21,3 +21,33 @@ const handleOptionButtonClick = (event) => {
 optionButtons.forEach((button) => {
   button.addEventListener('click', handleOptionButtonClick);
 });
+
+const randomizeAnswers = () => {
+  // grab and group multiple choice items by needs-js and no-js  pairs
+  const multipleChoiceItems = document.querySelectorAll(".cmp-multiple-choice__item");
+  const needsJSNoJSPairs = [];
+  for (const [index, element] of multipleChoiceItems.entries()) {
+    if (index % 2 === 0) {
+      needsJSNoJSPairs.push({
+        needsJS: element,
+        noJS: multipleChoiceItems[index + 1],
+      });
+    }
+  }
+  // randomize
+  const randomizedOrder = new Array(needsJSNoJSPairs.length);
+  needsJSNoJSPairs.forEach((pair) => {
+    let newPosition = -1;
+    do {
+      newPosition = Math.floor(Math.random() * (needsJSNoJSPairs.length));
+    } while (randomizedOrder[newPosition]);
+    randomizedOrder[newPosition] = pair;
+  });
+  // replace html
+  const multipleChoiceItemsContainer = document.querySelector('.cmp-multiple-choice');
+  multipleChoiceItemsContainer.innerHTML = '';
+  randomizedOrder.forEach(({needsJS, noJS}) =>{
+    multipleChoiceItemsContainer.append(needsJS, noJS);
+  });
```

Comment:
If you followed my other comments about ignoring the `noJS` elements, this would change to:

```js
randomizedOrder.forEach((element) => {
  multipleChoiceItemsContainer.append(element);
});
```